### PR TITLE
Prepare for v1.11.6+suite.1 release

### PR DESCRIPTION
**Pull request by izgeri**
In this suite release, the Conjur OpenAPI specification is added to the suite.

- [Wiki page](https://github.com/cyberark/conjur-oss-suite-release/wiki/v1.11.6-suite.1)
- [Draft release notes](https://gist.github.com/izgeri/eaffdff86d7505e640490c2eef28628b)

Draft What's new content:

---
## What's New

This Suite release aligns with Conjur Server [version 1.11.6](https://github.com/cyberark/conjur/releases/tag/v1.11.6). 
It includes the first linked release of the **Community Level** [Conjur OpenAPI v3 specification](https://github.com/cyberark/conjur-openapi-spec) with the Conjur server. The Conjur OpenAPI specification is an evolving project which provides a standardized, machine-readable version of the API that can be used to automatically generate API documentation, to generate Conjur client libraries in your language of choice, and to facilitate exploring the Conjur API using popular tools like Postman. 
This project simplifies end users’ ability to interact with the Conjur API, and will enable the continued expansion of Conjur integrations so that Conjur works natively with the most popular software development tools. We’ll be using this OpenAPI spec to build a better, more consistent, always up-to-date Conjur SDK comprised of clients in the most popular languages with comprehensive automated test coverage.  
For more information about the Conjur OpenAPI specification and how to use it, read our [recent blog post](https://www.conjur.org/blog/introducing-the-conjur-openapi-description/) on the topic.

For a reminder of the Conjur Open Source Suite Release structure please refer to the documentation [introduction](https://docs.conjur.org/Latest/en/Content/Overview/Conjur-OSS-Suite-Overview.html?#StructureofConjurOpenSourceSuite).

---

#### Change log
- [x] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [x] This PR includes new unit and integration tests to go with the code changes, or
- [ ] The changes in this PR do not require tests

#### Documentation
- [x] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [ ] This PR does not require updating any documentation

#### Release PRs
**If you are preparing for a release**, are the following items complete?
- [x] The PR title / description clearly state the suite release version.
- [ ] The [Jenkins dashboard](https://jenkins.conjur.net/view/OSS%20Suite%20Components/) shows no ongoing
      build failures for any included components.
- [x] The PR includes a link to a markdown release notes draft - this can be
     generated by running:
     ```
     summon -p keyring.py   --yaml 'GITHUB_TOKEN: !var github/api_token' \
       ./parse-changelogs \
       -v v1.11.6+suite.1 \
       -t release \
       -o RELEASE_NOTES.md
     ```
- [x] The PR includes PM-approved "What's new" text.

### CODE REVIEW

File: pkg/github/github.go [hunk 1]
```diff
@@ -275,14 +275,40 @@ func componentFromRepo(
 	// Check if there is a "releases/{suiteVersion}" branch
 	// If it exists, use that; if not, use master.
 	branch := "master"
-	hasReleaseBranch, err := checkForBranch(httpClient, "github_api", repo.Name, fmt.Sprintf("release/%s", suiteVersion))
+	hasReleaseBranch, err := checkForBranch(
```
Comment (doodlesbykumbi) [inline pkg/github/github.go:278]: The logic for determine the branch name is getting a bit busy so I think it'd be good for us to move that into it's own method.

File: suite.yml [hunk 2]
```diff
@@ -104,7 +108,7 @@ section:
         tool: Terraform
         description: Terraform provider that makes secrets in Conjur available in
           Terraform manifests.
-        version: v0.4.0
+        version: v0.5.0
```
[the remark this answers is not in the document]
Comment (izgeri) [reply suite.yml:111]: afaik, this is all that's required for 0.15 support. where do you think this documentation is required?

File: pkg/github/github.go (hunk 1 above)
Comment (izgeri) [reply pkg/github/github.go:278]: thanks @doodlesbykumbi - I'll take that on as a follow on to this PR. expect to be tagged as the reviewer!!

**Comment by boazmichaely** [comment]
@alexkalish @izgeri  
Regarding the Open API spec, it *is* a community level project and I think the warning is necessary. I'm in fact concerned with the WN wording and think the caveat is missing. I also miss the project objectives and I'm not finding where those are described. We discussed in the past that the OpenAPI has many benefits such as generate automatic documentation, generate the SDK. I feel it's important to set the stage for those objectives and make those benefits clear to all viewers.

on this line :
>with the Conjur server. Conjur Open Source users can now see the matching Conjur OpenAPI spec version on the

Can you please change to something like :
with the Conjur server.  **The Conjur OpenAPI specification is an evolving project { which will allow users to XXX | that has the following objectives ...}**  Users can currently see the matching Conjur OpenAPI spec version on the ...

File: suite.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (izgeri) [reply suite.yml:111]: Fixed up the Terraform provider README and filed https://github.com/cyberark/conjur-docs/issues/1588

[the remark this answers is not in the document]
Comment (alexkalish) [reply suite.yml:111]: The docs currently say:
image
So, should this be changed to make it clear that `sensitive` isn't optional?

**Review by alexkalish — APPROVED** [review]

**Review by boazmichaely — APPROVED** [review]
approved with the edited comment at the top, thanks !

File: suite.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (suefran) [reply suite.yml:111]: Just made this update - here is the [PR](https://app.zenhub.com/workspaces/aam-docs-5cf7a9ea1b8cc8533860c99c/issues/cyberark/conjur-docs/1591)

**Comment by izgeri** [comment]
I also made this change based on @boazmichaely's feedback: https://github.com/cyberark/conjur-openapi-spec/pull/190

**Comment by izgeri** [comment]
Note: I am merging this PR now to facilitate TW handoff, but we do have 2 current build failures to address by Friday:
- [Buildpack](https://jenkins.conjur.net/blue/organizations/jenkins/cyberark--conjur-api-ruby/detail/master/854/pipeline)
- [Conjur Ruby API](https://jenkins.conjur.net/blue/organizations/jenkins/cyberark--conjur-api-ruby/detail/master/854/pipeline)

**Pull request merged**